lib/gringotts/gateways/global_collect.ex

```diff
@@ -443,19 +421,36 @@ defmodule Gringotts.Gateways.GlobalCollect do
 
   defp respond({:ok, %{status_code: code, body: body}}) when code in [200, 201] do
     case decode(body) do
-      {:ok, results} -> {:ok, Response.success(raw: results, status_code: code)}
+      {:ok, results} ->
+        {
+          :ok,
+          Response.success(
+            authorization: results["payment"]["id"],
```

Comment:
Please populate other fields of the `Response` struct too, like `avs_result`, `cvv_result`, `message` and`fraud_result`